components/activity_log_modal/__snapshots__/activity_log_modal.test.jsx.snap

```diff
@@ -56,7 +56,7 @@ exports[`components/ActivityLogModal should match snapshot 1`] = `
       className="session-help-text"
     >
       <FormattedMessage
-        defaultMessage="Sessions are created when you log in to a new browser on a device. Sessions let you use Mattermost without having to log in again for a time period specified by the System Admin. If you want to log out sooner, use the 'Logout' button below to end a session."
+        defaultMessage="Sessions are created when you log in through a new browser on a device. Sessions let you use Mattermost without having to log in again for a time period specified by the system administrator. To end the session sooner, use the 'Log Out' button."
```

Comment:
@justinegeffen Wondering if you can weigh in on usage of ``System Admin`` vs ``system administrator``. I think we have mixed usage all over.  Also on lines 144 & line 230

Reply:
Thanks for catching that. In this case I'd definitely use "System Administrator" as that is more frequently used in the docs (and refers to a specific person rather than system administrators in general. 

Overall we do have a mixed bag, but generally we use caps for both System Admin and System Administrator. It would be ideal to agree on one and use that going forward. I think System Admin would work better (as would Team Admin) as we use "Admin" in the filter names, and a cursory glance at some PRs looks like we tend to refer to "Admin" and not "Administrator".

Reply:
If I could add my 2-cents, I'd suggest going with the lowercase 'system administrator'. Most techdocs use the initial caps for only company names and product names; even feature names aren't always capitalised unless they're sold as standalone stuff. Unless "System Administrator" or "System Admin" is an account type (like we have the 'Administrator' account type for Windows Server) , I'd bat for it being in all lowercase because, per my understanding, here it's just a role?

Reply:
0/5 but note that "System Admin" is an account type, similarly "Team Admin". It refers to an account that has permissions to access certain features as defined by permission schemes

Reply:
Changed 'system administrator' to 'System Admin'.